src/main/PersistentState.cpp

```diff
@@ -78,7 +80,8 @@ PersistentState::getSCPStateAllSlots()
     std::vector<std::string> states;
     for (uint32 i = 0; i <= mApp.getConfig().MAX_SLOTS_TO_REMEMBER; i++)
     {
-        auto val = getFromDb(getStoreStateName(kLastSCPData, i));
+        auto val = states.emplace_back(
```

Comment:
I think this got borked: it should just be `auto val = getFromDb(getStoreStateName(kLastSCPData, i));`

Reply:
Thanks for catching this, fixed.